packages/polaris-viz/src/components/SimpleNormalizedChart/stories/SimpleNormalizedBarChart.chromatic.stories.tsx

```diff
@@ -0,0 +1,60 @@
+import {storiesOf} from '@storybook/react';
```

Comment:
@envex something like this?

Reply:
https://6062ad4a2d14cd0021539c1b-kkkyfnhjos.chromatic.com/?path=/story/chromatic-components--simplenormalizedchart 

does this help?

Reply:
Yeah, I can still view the chromatic tests, I just can't login to set it to an approved change.

Reply:
oh okay 👍🏻